doc/man7/EVP_KDF-SSHKDF.pod

```diff
@@ -41,7 +41,7 @@ These parameters work as described in L<EVP_KDF(3)/PARAMETERS>.
 These parameters set the respective values for the KDF.
 If a value is already set, the contents are replaced.
 
-=item "type" (B<OSSL_KDF_PARAM_SSHKDF_TYPE>) <integer>
+=item "type" (B<OSSL_KDF_PARAM_SSHKDF_TYPE>) <UTF8 string>
 
 This parameter sets the type for the SSHHKDF operation.
```

Comment:
Please fix this typo SSHHKDF when you're at it.

Reply:
Done